python_modules/dagster/dagster_tests/storage_tests/utils/run_storage.py

```diff
@@ -1036,8 +1037,9 @@ def test_fetch_run_group(self, storage: RunStorage):
 
         assert len(run_group_two[1]) == 7
 
-        assert run_group_one[0] == run_group_two[0]
-        assert run_group_one[1] == run_group_two[1]
+        # The order of runs in each run run group is not deterministic
```

Comment:
I'm just commenting what I observed - if we feel the order should be sorted, I can enforce that behavior instead (or dig deeper into why it's not happening if it's already expected).

Reply:
maybe include something like  "at this time `get_run_group` does not apply an ordering"